backend/src/seeds/seed.user.ts

```diff
@@ -46,6 +47,10 @@ class UserFactory extends Factory<User> {
       user.trainee = traineesFactory.generate();
     }
 
+    if (userType === 'instructor') {
+      user.instructor = instructorsFactory.generate();
+    }
+
     this.entities.push(user);
 
     return user;
```

Comment:
This is regarding method `generateFromData`
1. After adding `null` to `trainee` field in the `User` entity TypeScript will throw an error. To fix it you have to add `!` after the `user.trainee`
```ts
if (trainee === null) {
  traineesFactory.remove(user.trainee!);
} else if (trainee !== undefined) {
  user.trainee = trainee;
}
```
2. We should also add this for the instructor now (even though we are not using this directly). This is essentially a bit of a hacky way to prevent creating duplicate trainees/instructors as they are being automatically generated in the `this.generate` (which is used underneath `generateFromData`, please see [backend/src/seeds/seed.utils.ts#L22](https://github.com/twoje-osk/twoje-osk/blob/main/backend/src/seeds/seed.utils.ts#L22))